spec/definitions/boxes/definitions/base_ubuntu14.04_kvm/definition.rb

```diff
@@ -0,0 +1,45 @@
+Veewee::Session.declare({
+  :cpu_count => '1',
+  :memory_size => '512',
+  :disk_size => '12040',
+  :disk_format => 'VDI',
+  :hostiocache => 'off',
+  :os_type_id => 'Ubuntu_64',
+  :iso_file => "ubuntu-14.04.3-server-amd64.iso",
+  :iso_src => "http://releases.ubuntu.com/14.04/ubuntu-14.04.3-server-amd64.iso",
+  :iso_md5 => "9e5fecc94b3925bededed0fdca1bd417",
+  :iso_download_timeout => "1000",
+  :boot_wait => "4",
+  :boot_cmd_sequence => [
+    '<Esc><Esc><Enter>',
+    '/install/vmlinuz noapic preseed/url=http://%IP%:%PORT%/preseed.cfg ',
+    'debian-installer=en_US auto locale=en_US kbd-chooser/method=us ',
+    'hostname=base-ubuntu-14.04 ',
+    'fb=false debconf/frontend=noninteractive ',
+    'keyboard-configuration/modelcode=SKIP keyboard-configuration/layout=us keyboard-configuration/variant=us console-setup/ask_detect=false ',
+    'initrd=/install/initrd.gz -- <Enter>'
+],
+  :kickstart_port => "7122",
+  :kickstart_timeout => "300",
+  :kickstart_file => "preseed.cfg",
+  :ssh_login_timeout => "10000",
+  :ssh_user => "vagrant",
+  :ssh_password => "vagrant",
+  :ssh_key => "",
+  :ssh_host_port => "7222",
+  :ssh_guest_port => "22",
+  :sudo_cmd => "echo '%p'|sudo -S sh '%f'",
+  :shutdown_cmd => "shutdown -P now",
+  :postinstall_files => [
+   "build_time.sh",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.